compiler/lib/src/main/scala/analysis/Semantics/StateMachineInstance.scala

```diff
@@ -0,0 +1,28 @@
+package fpp.compiler.analysis
+
+import fpp.compiler.ast._
+import fpp.compiler.util._
+
+/** An FPP data product containter */
```

Comment:
Looks like a stray comment: "data product container" should be "state machine instance"

Reply:
Fixed